### ISSUE TITLE
undefined method `strftime'

### ISSUE THREAD

**Issue by Andreas-Halemba** [fastlane/fastlane#8271]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

/Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/match/lib/match/nuke.rb:99:in `block in print_tables': [!] undefined method `strftime' for "2017-10-31T13:34:55Z":String (NoMethodError)
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/match/lib/match/nuke.rb:96:in `collect'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/match/lib/match/nuke.rb:96:in `print_tables'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/match/lib/match/nuke.rb:23:in `run'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/match/lib/match/commands_generator.rb:111:in `block (3 levels) in run'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-4.4.3/lib/commander/command.rb:178:in `call'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-4.4.3/lib/commander/command.rb:178:in `call'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-4.4.3/lib/commander/command.rb:153:in `run'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-4.4.3/lib/commander/runner.rb:446:in `run_active_command'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:38:in `run!'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/commander-4.4.3/lib/commander/delegates.rb:15:in `run!'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/match/lib/match/commands_generator.rb:118:in `run'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/match/lib/match/commands_generator.rb:12:in `start'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/fastlane/lib/fastlane/cli_tools_distributor.rb:49:in `take_off'
	from /Users/andreas/.fastlane/bin/bundle/lib/ruby/gems/2.2.0/gems/fastlane-2.17.0/bin/fastlane:15:in `<top (required)>'
	from /Users/andreas/.fastlane/bin/bundle/bin/fastlane:22:in `load'
	from /Users/andreas/.fastlane/bin/bundle/bin/fastlane:22:in `<main>'

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                 | Value                                       |
| ------------------- | ------------------------------------------- |
| OS                  | 10.12.3                                     |
| Ruby                | 2.2.4                                       |
| Bundler?            | false                                       |
| Git                 | git version 2.10.1 (Apple Git-78)           |
| Installation Source | ~/.fastlane/bin/bundle/bin/fastlane         |
| Host                | Mac OS X 10.12.3 (16D32)                    |
| Ruby Lib Dir        | ~/.fastlane/bin/bundle/lib                  |
| OpenSSL Version     | OpenSSL 1.0.2g  1 Mar 2016                  |
| Is contained        | true                                        |
| Is homebrew         | false                                       |
| Xcode Path          | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version       | 8.2.1                                       |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version '1.99.0'

default_platform :ios

platform :ios do
    before_all do
        # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
        ENV['FL_HIPCHAT_MESSAGE'] = 'Build in fastlane is done and uploaded to Testflight'
        ENV['FL_HIPCHAT_CHANNEL'] = 'Fastlane'
        ENV['HIPCHAT_API_TOKEN'] = 'MCKdlarFJeVElNPfYmEL2fyTZJKBzcBlQnDvgaSf'
        ENV['FL_HIPCHAT_CUSTOM_COLOR'] = 'green'
        ENV['HIPCHAT_NOTIFY_ROOM'] = 'true'
        ENV['HIPCHAT_API_VERSION'] = '2'
    end

    desc 'Runs all the tests'
    lane :testChat do
        hipchat
    end

    desc 'test bashscript'
    lane :bashTest do
        sh './runGrunt.sh'
    end

    desc 'Submit a new Beta Build to Apple TestFlight'
    desc 'This lane is to upload a Jaz whitelable app.'
    desc 'This will also make sure the profile is up to date'
    lane :betaJaz do
        sh 'cd ../../../.. && grunt switchWhitelabel --hotelName=jaz && grunt compass && grunt pg:deploy:ios --Verbose'
        match(type: 'appstore') # more information: https://codesigning.guide
        increment_build_number
        gym(scheme: 'jaz') # Build your app - more options available
        deliver(force: true, verbose: true)
        hipchat
    end

    desc 'Submit a new Beta Build to Apple TestFlight'
    desc 'This lane is to upload a hotelbird app.'
    desc 'This will also make sure the profile is up to date'
    lane :betaHbapp do
        sh 'cd ../../../.. && grunt switchWhitelabel --Verbose && grunt compass && grunt pg:deploy:ios --Verbose'
        match(type: 'appstore') # more information: https://codesigning.guide
        increment_build_number
        gym(scheme: 'hotelbird') # Build your app - more options available
        deliver(force: true, verbose: true)
        hipchat
    end

    desc 'Deploy a new version to the App Store'
    lane :appstoreUpload do
        # match(type: "appstore")
        # snapshot
        gym # Build your app - more options available
        deliver(scheme: 'Jaz', skip_metadata: true, skip_screenshots: true, force: true, verbose: true)
        # frameit
    end

    # You can define as many lanes as you want

    after_all do |lane|
        # This block is called, only if the executed lane was successful

        # slack(
        #   message: "Successfully deployed new App Update."
        # )
    end

    error do |lane, exception|
        ENV['FL_HIPCHAT_MESSAGE'] = 'Lane in fastlane is faild.'
        ENV['FL_HIPCHAT_CHANNEL'] = 'Fastlane'
        ENV['FL_HIPCHAT_CUSTOM_COLOR'] = 'red'
    end
end

# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md

# fastlane reports which actions are used
# No personal data is recorded. Learn more at https://github.com/fastlane/enhancer
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
# app_identifier "com.hotelbird.hb-white-label-app" # The bundle identifier of your app
# apple_id "andreas@hotelbird.de" # Your Apple email address
team_id "A9EF3YCJ4H"  # Developer Portal Team ID

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Appfile.md
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.17.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| CFPropertyList            | 2.3.5        |
| activesupport             | 4.2.7.1      |
| addressable               | 2.5.0        |
| babosa                    | 1.0.2        |
| bigdecimal                | 1.2.6        |
| bundler                   | 1.14.3       |
| claide                    | 1.0.1        |
| colored                   | 1.2          |
| commander                 | 4.4.3        |
| domain_name               | 0.5.20161129 |
| dotenv                    | 2.2.0        |
| excon                     | 0.54.0       |
| faraday                   | 0.11.0       |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.11.0.1     |
| fastimage                 | 2.0.1        |
| gh_inspector              | 1.0.3        |
| google-api-client         | 0.9.26       |
| googleauth                | 0.5.1        |
| highline                  | 1.7.8        |
| http-cookie               | 1.0.3        |
| httpclient                | 2.8.3        |
| hurley                    | 0.2          |
| i18n                      | 0.7.0        |
| io-console                | 0.4.3        |
| json                      | 1.8.1        |
| jwt                       | 1.5.6        |
| libxml-ruby               | 2.9.0        |
| little-plugger            | 1.1.4        |
| logging                   | 2.1.0        |
| memoist                   | 0.15.0       |
| mime-types                | 3.1          |
| mime-types-data           | 3.2016.0521  |
| mini_magick               | 4.5.1        |
| minitest                  | 5.4.3        |
| multi_json                | 1.12.1       |
| multi_xml                 | 0.6.0        |
| multipart-post            | 2.0.0        |
| nanaimo                   | 0.2.3        |
| nokogiri                  | 1.5.6        |
| os                        | 0.9.6        |
| plist                     | 3.2.0        |
| power_assert              | 0.2.2        |
| psych                     | 2.0.8        |
| public_suffix             | 2.0.5        |
| rake                      | 10.4.2       |
| rdoc                      | 4.2.0        |
| representable             | 2.3.0        |
| retriable                 | 2.1.0        |
| rouge                     | 1.11.1       |
| rubygems-update           | 2.5.2        |
| rubyzip                   | 1.2.0        |
| security                  | 0.1.3        |
| signet                    | 0.7.3        |
| slack-notifier            | 1.5.1        |
| sqlite3                   | 1.3.13       |
| terminal-notifier         | 1.7.1        |
| terminal-table            | 1.7.3        |
| test-unit                 | 3.0.8        |
| thread_safe               | 0.3.5        |
| tty-screen                | 0.5.0        |
| tzinfo                    | 1.2.2        |
| uber                      | 0.0.15       |
| unf                       | 0.1.4        |
| unf_ext                   | 0.0.7.2      |
| unicode-display_width     | 1.1.3        |
| word_wrap                 | 1.0.0        |
| xcodeproj                 | 1.4.2        |
| xcpretty                  | 0.2.4        |
| xcpretty-travis-formatter | 0.0.4        |
</details>

*generated on:* **2017-02-16**
</details>

**Comment by lacostej** [comment]
This should also be fixed by #8270

**Comment by lacostej** [comment]
Closing issue as this is a duplicate of #8266
